### PR TITLE
Add Kafka authentication and authorization description in TiCDC

### PR DESCRIPTION
**Pull request by zhaoxinyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -232,6 +241,55 @@ URI 中可配置的的参数如下：
...
+
+SASL/GSSAPI `keytab` 类型认证：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="kafka://127.0.0.1:9092/topic-name?kafka-version=2.4.0&sasl-mechanism=gssapi&sasl-gssapi-auth-type=keytab&sasl-gssapi-kerberos-config-path=/etc/krb5.conf&sasl-gssapi-service-name=a&sasl-gssapi-user=alice/for-kafka&sasl-gssapi-keytab-path=/var/lib/secret/alice.key&sasl-gssapi-realm=example.com"
+```
+
+SASL/GSSAPI 认证方式详见 [Configuring GSSAPI](https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_gssapi.html)
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:280]:
```suggestion
SASL/GSSAPI 认证方式详见 [Configuring GSSAPI](https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_gssapi.html)。
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -232,6 +241,55 @@ URI 中可配置的的参数如下：
...
+- TLS/SSL 加密
+
+如果 Kafka broker 启用了 TLS/SSL 加密，则需要在 `--sink-uri` 中增加 `enable-tls=true` 参数值。如果需要使用自签名证书，则还需要在 `--sink-uri` 中指定 `ca`、`cert` 跟 `key` 几个参数。
+
+- ACL 授权
+
+TiCDC 能够正常工作所需的最小权限集合如下：
+
+1. 对 Topic [资源类型](https://docs.confluent.io/platform/current/kafka/authorization.html#resources)的 `Create` 和 `Write` 权限
+2. 对 Cluster 资源类型的 `DescribeConfigs` 权限。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:291]:
```suggestion
- 对 Topic [资源类型](https://docs.confluent.io/platform/current/kafka/authorization.html#resources)的 `Create` 和 `Write` 权限。
- 对 Cluster 资源类型的 `DescribeConfigs` 权限。
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -574,7 +632,7 @@ worker-num = 16
 
 [sink]
 # 对于 MQ 类的 Sink，可以通过 dispatchers 配置 event 分发器
-# 支持 partition 及 topic 两种 event 分发器。二者的详细说明见下一节。
+# 支持 partition 及 topic（从 v6.1 开始支持）两种 event 分发器。二者的详细说明见下一节。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:635]: 这句话的英文放到这个 PR 里了： https://github.com/pingcap/docs/pull/8721/files

Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:635]: > 这句话的英文放到这个 PR 里了： https://github.com/pingcap/docs/pull/8721/files

topic 分发器是从 v6.1 开始的，partition 分发器的版本还需要提及吗 @leoppro

Comment (leoppro) [reply ticdc/manage-ticdc.md:635]: 不用保持现状即可。文档的目的是让人理解，不需要格式的完美对称

**Review by leoppro — APPROVED** [review]
LGTM

**Review by hi-rustin — APPROVED** [review]

**Review by zhangyangyu — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**